decentraland/unity-explorer
# chore: load thumbnails through ABs

## What does this PR change?

- Adds the possibility to load thumbnails through AB. This is going to allow image caching, therefore not downloading it everytime
- Adds a fallback for the regular texture request, in case the manifest is not avaialble
- Adds some cleaning to the process. It doesnt make sense to request the thumbnail unless its going to be displayed

## How to test the changes?

1. Launch the explorer
2. Check thumbnails are displayed correctly everywhere (backpack wearables, backpack emotes, change pages, emotes wheel)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
